wrappers/awskms/awskms.go

```diff
@@ -52,11 +52,18 @@ type Wrapper struct {
 
 	currentKeyId *atomic.Value
 
-	client kmsiface.KMSAPI
+	client KmsApi
 
 	logger hclog.Logger
 }
 
+// Replaces aws-sdk v1's iface package interfaces
```

Comment:
Nit: this comment neatly explains why this change was made in this PR, but it doesn't really explain to the person reading this in a year or two's time why we have this interface. What do you think about something like
```suggestion
// KmsApi defines the functionality expected to be implemented by the AWS SDK v2 kms package.
```